pkg/dbt_cloud/job.go

```diff
@@ -59,26 +59,28 @@ type JobCompletionTriggerCondition struct {
 
 type Job struct {
 	ID                     *int                  `json:"id"`
-	Account_Id             int                   `json:"account_id"`
-	Project_Id             int                   `json:"project_id"`
-	Environment_Id         int                   `json:"environment_id"`
+	AccountId              int                   `json:"account_id"`
+	ProjectId              int                   `json:"project_id"`
+	EnvironmentId          int                   `json:"environment_id"`
 	Name                   string                `json:"name"`
+	DbtVersion             *string               `json:"dbt_version"`
```

Comment:
Same here. Did you sort them by alphabetical order or are you using a tool for it? Historically, I have been trying to group new fields together but it also makes sense to have those in alphabetical order.

Reply:
The ordering I did by hand using IntelliJ to move the lines up/down (cmd-shift up/down).